### DISCUSSION TITLE
Notification Center Bell Icon

### DISCUSSION THREAD

**Discussion by NjlsShade** [linuxdeepin/developer-center#3205] [Features Request & Ideas | 特性请求 & 头脑风暴]
### SUMMARY | 问题概要

There is a "Bell Icon" in the top left of the Notification Center, it's been there for a very long time and I'm still yet to understand what it does. I assume it would be used to mute notifications, which would be a nice shortcut for that, however, it does not have this functionality currently.

What was this intended to do?
![Screenshot_select-area_20220528155055](https://user-images.githubusercontent.com/15788214/170841240-abc2a7ad-26be-48cc-9e24-6aead060c1ce.png)

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

Using the latest deepin 20.5.

### ADDITIONAL INFORMATION | 额外补充

Issue [2542](https://github.com/linuxdeepin/developer-center/issues/2542) brought it up as well, but was closed without a clear indication on what the icon is used for, rather just provided an alternate method to mute notifications. I think this icon should toggle "Do Not Disturb" mode.

**Comment by ghost** [comment]
Also, in my opinion, notifications stays too long into Notification Center when not dismiss them. Would be nice after 1 day they dismiss automatically or provide a custom time to users. I have built up notifications from up to a week ago and I can’t find anymore what is useful or what was label for

**Comment by BLumia** [comment]
I guess it does nothing currently, it's just an icon that indicates the panel is related to notification.

@meiyixiang : 目前的设计，通知区域左上角的图标，用户总会以为能点击（点击的时候甚至有变色效果，用户期望点击交互会有对应的行为，比如切换勿扰模式），但实际并不能点击。希望能确认相关的交互行为，看看是否需要调整。

**Reply by kt286** [reply]
我就被这个设计坑过

**Comment by meiyixiang** [comment]
Thanks for your advice. I have communicated with relevant products about this problem a long time ago, and this V20 version should not be optimized for this problem. Now I am developing the V23 version, and I will avoid such problems in the new desktop design
